Added whois and email functionality

ryansmick opened the pull request:
Implemented a Django app to determine the owner of a website and
send that owner an email

ryansmick committed:
Added whois and email functionality
Implemented a Django app to determine the owner of a website and
send that owner an email

dmattia commented on edge_cms_mail/owner_conf/views.py:
```diff
@@ -0,0 +1,37 @@
+				Thank you for using edge-cms on your domain: {}'''.format(owner, domain),
+			"DoNotReply@edge-cms.com",
+			[email],
+			fail_silently=True)
```
Could you expand on why you have fail_silently = True as well as a check for smtplib.SMTPException?

ryansmick replied:
Whoops, I meant to have it false. I'll make all of the changes and resubmit.

dmattia commented on edge_cms_mail/edge_cms_mail/settings.py:
```diff
@@ -0,0 +1,120 @@
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
```
can you add owner_conf to installed_apps

dmattia commented on edge_cms_mail/edge_cms_mail/settings.py:
```diff
@@ -0,0 +1,120 @@
+# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '$$35*a6fj60t!1_n7#1qrvw^z39e*8iaf9$k&i#*v=^8f!6e(i'
```
Use an environment variable if possible, or generate a "testing" key, or add the settings file to the .gitignore

dmattia commented on edge_cms_mail/manage.py:
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import os
```
Would be nice to have a .gitignore file with *.pyc to remove compiled python

dmattia commented on edge_cms_mail/owner_conf/views.py:
```diff
@@ -0,0 +1,37 @@
+		send_mail(
+			"Please Confirm Email Address",
+			'''Hello {},
+				Thank you for using edge-cms on your domain: {}'''.format(owner, domain),
```
possible extra tab

ryansmick committed:
Made Appropriate Changes
Added .gitignore, removed .pyc files, fixed fail_silently flag, removed
secret key from settings.py, and added owner_conf to installed apps

ryansmick commented:
I added the new changes in the "Made Appropriate Changes" commit, which should be included in this pull request.
